packages/revocation/package.json

```diff
@@ -35,7 +35,7 @@
     "@ew-did-registry/keys": "0.6.0",
     "@ew-did-registry/did-ethr-resolver": "0.6.0",
     "@energyweb/iam-contracts": "3.3.0",
-    "ethers": "^5.4.7"
+    "ethers": "5.4.7"
```

Comment:
Why not updating the version in the `revocation` package?

Reply:
There was some dependencies problem when I last time tried to update it. Ashish said we can leave it as it is because revocation anyways will be moved to `ew-credentials`. I'll try to update it though